test/testError/benchmarkWithError.sh

```diff
@@ -1,19 +1,13 @@
 #!/bin/bash
 
+# This benchmark is used in errorTest.sh.
+
 IMAGES="Pharo11"
 VMs="latest10"
-PHARO_CMD="nil"
-
-runBenchs
-
-PHARO_CMD="eval 1+1"
-
-runBenchs
-
-PHARO_CMD="nil"
-
-runBenchs
+ITERATIONS=4
 
-PHARO_CMD="eval 1+1"
+# This command only fails if ITERATION is odd, so that with 4 iterations, it fails one in two times.
+# In errorTest it is checked that the iterations indeed failed one in two times
+PHARO_CMD='eval ($ITERATION \\ 2) == 0 ifFalse: [ self error ]'
```

Comment:
I wouldn't replace this test, I would make a new one.
Because they both test different things and both are important.

Reply:
Yes :) Just recover the old code and have two separate files with different names :)

Also, choose more intention revealing names for files :)